### DISCUSSION TITLE
Google Consent Mode

### DISCUSSION THREAD

**Discussion by Bazoul54** [AmauriC/tarteaucitron.js#929] [General]
Bonjour AmauriC,

Un grand merci pour Tarte au Citron que j'utilise sur beaucoup de sites.

Je n'ai pas suffisamment de compétence, et j'ai besoin de ton aide, car je veux rester en conformité avec le RGPD mais également pouvoir continuer à avoir de bons résultats sur les campagnes Google Ads de mes clients grâce aux conversions.

Je viens d'avoir un rendez-vous téléphonique avec un technicien de Google ads, il m'a parlé du "Google Consent Mode". (https://developers.google.com/tag-platform/devguides/consent)

Il me l'a mis en place sur Tag Manager avec un fichier Json.

Je suis incapable de reproduire ce qu'il a fait, et je me demandais si il y avait une implémentation de prévue sur TAC "plus simple".

Je peux te fournir le fichier si besoin.

Je te remercie d'avance pour ta réponse.

Cordialement, Bazoul.

**Comment by AmauriC** [comment]
Bonjour : )

Par curiosité je veux bien le fichier json mais je ne pense pas que tac puise être compatible avec le "consent mode" de google qui est surtout un moyen de passer outre la réglementation : /

**Reply by Bazoul54** [reply]
Ah mince, si ce n est pas conforme, ça craint !

Le type de google connaissait TAC ! Et il l a dit pas de problème pour google consent.

Je t ai envoyé un mail avec le fichier.

**Reply by AmauriC** [reply]
Merci 👌

Apparement ça check si dans le cookie tarteaucitron `marketing=true!` est présent pour gérer le consent mode.
A moins d'avoir ajouté manuellement un service "marketing", tarteaucitron ne le gère pas. 

Du coup pas de vraiment comprendre comment ça fonctionne ^^

**Reply by Bazoul54** [reply]
Je m aperçoit que de toute façon ça ne fonctionne pas, car les cookies sont chargés à l ouverture du site.

Je vais revenir à la config que j utilise habituellement (avec la barre de chargement au scroll au dessus du bandeau des cookies)

Merci pour tes réponses !

**Comment by CaroS77** [comment]
Bonjour,

Je vous félicite pour votre travail et je me permets de relancer cette conversation. 
Avez-vous pu effectuer des modifcations sur ce sujet ? 
Nous avons également contacter Google qui nous indique qu'il est possible de mettre en place le mode consent dans TAC. 
Est-ce vrai ou pas ? Nous passons par le module WP.

Après je comprendrais très bien qu'ils vendent leur produit et qu'ils nous ont communiqué de mauvaises infos concernant la mise en place de ce consent dans votre module.

Je vous remercie d'avance, si vous pouviez m'informer sur le sujet.

Merci beaucoup

**Comment by AmauriC** [comment]
Salut,

Toujours pas de changement.

Pour que ça fonctionne comme veux Google, il faudrait de toute façon laisser les tags Google se charger dès le départ et faire confiance à Google pour respecter la vie privé de l'utilisateur tant qu'il n'a pas dit oui via son système de consent, ce n'est pas autorisé par la cnil et ça ne le sera probablement jamais : /

**Comment by CaroS77** [comment]
Ok, merci pour ce retour Amauri.
Bon, l'idéal est donc vraiment de passer par une interface d'analyse français, à suivre...
Merci bien,
